[b-8] Add movie review

andreyryzhkov opened the pull request:
1) Insert new review to database in DAO
2) Added ReviewService
3) Added ReviewController for storing review
2) Added test

andreyryzhkov committed:
[b-8] Add movie review

1) Insert new review to database in DAO
2) Added ReviewService
3) Added ReviewController for storing review
2) Added test

Rooman commented on src/main/java/com/aryzhkov/movieland/web/controller/ReviewController.java:
```diff
@@ -0,0 +1,46 @@
+    private final SecurityService securityService;
+
+    @PostMapping(path = "/review")
+    public void addReview(@RequestHeader(value = "token") String token,
```
@ProtectedBy(UserRole.USER)

Rooman replied:
Add spring incerceptor

Rooman replied:
https://github.com/AnabolKiev/movieland/pull/22/files check review comments

andreyryzhkov committed:
[b-8] Fixed after review

1) Created AuthInterceptor
2) Created annotation ProtectedBy
3) Re-factor ReviewController

andreyryzhkov committed:
[b-8] Fixed after review

1) Removed isAuthorized

andreyryzhkov committed:
Merge remote-tracking branch 'remotes/origin/master' into b_9_add_movie_review

# Conflicts:
#	src/main/java/com/aryzhkov/movieland/security/SecurityService.java

andreyryzhkov committed:
b-8 Fixed after review

andreyryzhkov committed:
b-8 Fixed after review

1) Remove expiring session
